src/prommis/uky/uky_flowsheet.py

```diff
@@ -862,5 +874,719 @@ def display_results(m):
+        "Sc2O3": 0.00143 * units.kg / units.hr,
+        "Y2O3": 0.05418 * units.kg / units.hr,
+        "La2O3": 0.13770 * units.kg / units.hr,
+        "CeO2": 0.37383 * units.kg / units.hr,
+        "Pr6O11": 0.03941 * units.kg / units.hr,
+        "Nd2O3": 0.17289 * units.kg / units.hr,
+        "Sm2O3": 0.02358 * units.kg / units.hr,
+        "Eu2O3": 0.00199 * units.kg / units.hr,
+        "Gd2O3": 0.00000 * units.kg / units.hr,
+        "Tb4O7": 0.00801 * units.kg / units.hr,
+        "Dy2O3": 0.00000 * units.kg / units.hr,
+        "Ho2O3": 0.00000 * units.kg / units.hr,
+        "Er2O3": 0.00000 * units.kg / units.hr,
+        "Tm2O3": 0.00130 * units.kg / units.hr,
+        "Yb2O3": 0.00373 * units.kg / units.hr,
+        "Lu2O3": 0.00105 * units.kg / units.hr,
+    }
+
+    m.fs.costing.build_process_costs(
+        # arguments related to installation costs
+        piping_materials_and_labor_percentage=20,
+        electrical_materials_and_labor_percentage=20,
+        instrumentation_percentage=8,
+        plants_services_percentage=10,
+        process_buildings_percentage=40,
+        auxiliary_buildings_percentage=15,
+        site_improvements_percentage=10,
+        equipment_installation_percentage=17,
+        field_expenses_percentage=12,
+        project_management_and_construction_percentage=30,
+        process_contingency_percentage=15,
+        # argument related to Fixed OM costs
+        nameplate_capacity=500,  # short (US) ton/hr
+        labor_types=[
+            "skilled",
+            "unskilled",
+            "supervisor",
+            "maintenance",
+            "technician",
+            "engineer",
+        ],
+        labor_rate=[24.98, 19.08, 30.39, 22.73, 21.97, 45.85],  # USD/hr
+        labor_burden=25,  # % fringe benefits
+        operators_per_shift=[4, 9, 2, 2, 2, 3],
+        hours_per_shift=hours_per_shift,
+        shifts_per_day=shifts_per_day,
+        operating_days_per_year=operating_days_per_year,
+        pure_product_output_rates=pure_product_output_rates,
+        mixed_product_output_rates=mixed_product_output_rates,
+        mixed_product_sale_price_realization_factor=0.65,  # 65% price realization for mixed products
+        # arguments related to total owners costs
+        land_cost=m.fs.land_cost,
+        resources=resources,
+        rates=rates,
+        fixed_OM=True,
+        variable_OM=True,
+        feed_input=m.fs.feed_input,
+        efficiency=0.80,  # power usage efficiency, or fixed motor/distribution efficiency
+        waste=[
+            "nonhazardous_solid_waste",
+            "nonhazardous_precipitate_waste",
+            "dust_and_volatiles",
+        ],
+        recovery_rate_per_year=m.fs.recovery_rate_per_year,
+        CE_index_year=CE_index_year,
+    )
+
+    # define reagent fill costs as an other plant cost so framework adds this to TPC calculation
+    m.fs.costing.other_plant_costs.unfix()
+    m.fs.costing.other_plant_costs_rule = Constraint(
```

Comment:
These would scale down or import from the flowsheet as well, they are the cost of pre-filling these unit operations with chemicals prior to start-up. If the chemicals exist in the dictionary [here](https://github.com/prommis/prommis/blob/b903c34908f76c6dd6936a13d41baa64f45a12fd/src/prommis/uky/costing/ree_plant_capcost.py#L2048), you can use those prices to estimate the absolute costs and update these values.